multi-node/aws/pkg/cluster/config.go

```diff
@@ -66,8 +120,11 @@ func DecodeConfigFromFile(out *Config, loc string) error {
 
 func NewDefaultConfig(ver string) *Config {
 	return &Config{
-		ClusterName: "kubernetes",
-		ArtifactURL: DefaultArtifactURL(ver),
+		ClusterName:  "kubernetes",
+		ArtifactURL:  DefaultArtifactURL(ver),
+		VpcCIDR:      DefaultVpcCIDR,
+		SubnetCIDR:   DefaultSubnetCIDR,
+		ControllerIP: DefaultControllerIP,
```

Comment:
I do it so I don't have to check for empty string for every key in the validation step. Has the nice side effect of validating the defaults.